cucumber/api/features/step_definitions/logs_steps.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Given(/^I save the amount of log lines into "([^"]*)"$/) do |bookmark|
```

Comment:
this step name isn't clear. can we change it to "I save the log data from bookmark (([^"]*)"?